### DISCUSSION TITLE
Repeater second field value stays the same

### DISCUSSION THREAD

**Discussion by koneubauer** [Budibase/budibase#11072] [Help]
I am using a selfhosted image of 2.7.33

I have a dataprovider which provides two columns: name and number. When putting a table into the data provider both columns and their rows are shown correctly.
However, when i put a repeater inside and put a number field in that, the default value, which i have set to be the number is the same in every field. I set the name column as the label of the number field and that works. It's just that the number is always 5 in this case

**Comment by koneubauer** [comment]
When i put the same handelbars notation in the Placeholder field, it works.

**Comment by koneubauer** [comment]
When using the built-in DevTools, under Choose Component, the default value is correct, however in the UI it's wrong

**Comment by melohagan** [comment]
Hey @koneubauer 

Screenshots would be helpful in this case. It'd be nice to see your component tree and the bindings you are using
